src/core/client/auth/mutations/SignOffMutation.ts

```diff
@@ -0,0 +1,28 @@
+import { commitLocalUpdate, Environment } from "relay-runtime";
+import { TalkContext } from "talk-framework/lib/bootstrap";
+import { createMutationContainer } from "talk-framework/lib/relay";
+import { LOCAL_ID } from "talk-framework/lib/relay/withLocalStateContainer";
+
+export type SignOffMutation = () => Promise<void>;
+
+export async function commit(
+  environment: Environment,
```

Comment:
The second argument must be the input even if its empty or undefined, the context is passed as the third argument.